proto/nftmarket/query.proto

```diff
@@ -120,6 +128,35 @@ message QueryNftBidsRequest {
 message QueryNftBidsResponse {
   repeated NftBid bids = 1 [(gogoproto.nullable) = false];
 }
+message QueryPaymentStatusRequest {
+  string class_id = 1;
+  string nft_id   = 2;
+  string bidder   = 3 [
+    (gogoproto.moretags)  = "yaml:\"bidder\"",
+    (cosmos_proto.scalar) = "github.com/UnUniFi/chain/types.StringAccAddress"
+  ];
+}
+
+message QueryPaymentStatusResponse {
+  PaymentStatus paymentStatus = 1 [(gogoproto.nullable) = false];
+}
+
+import "google/protobuf/timestamp.proto";
```

Comment:
I think import should be on top not in the middle.

Reply:
fixed